BackofficeBundle/Resources/views/Form/form_div_layout.html.twig

```diff
@@ -1,5 +1,18 @@
 {% extends 'BraincraftedBootstrapBundle:Form:bootstrap.html.twig' %}
 
+{% block form %}
+    {% spaceless %}
+        {% if false == form.vars.valid %}
+            <div class="alert alert-danger">
+                There are errors in the form
```

Comment:
I didn't took the time to do it :). I was more interested with the comment on how I did this :)